core/src/test/java/nu/yona/server/messaging/service/MessageServiceTest.java

```diff
@@ -167,18 +173,34 @@ public void sendMessage_anonymousDestination_sendsFirebaseMessage()
 	}
 
 	@Test
-	public void sendMessage_namedDestination_doesNotSendFirebaseMessage()
+	public void sendDirectMessage_default_doesNotSendFirebaseMessageButSendsSms()
 	{
 		Message message = Mockito.mock(Message.class);
-
 		MessageDestination namedMessageDestination = Mockito.mock(MessageDestination.class);
+		String mobileNumber = "+316123456789";
+		User user = Mockito.mock(User.class);
+		when(user.getNamedMessageDestination()).thenReturn(namedMessageDestination);
+		when(user.getMobileNumber()).thenReturn(mobileNumber);
+
+		service.sendDirectMessage(message, user);
 
+		verify(mockFirebaseService, never()).sendMessage(any(), any());
```

Comment:
If it's worth verifying that direct messages don't push through Firebase, I think we should also verify no SMS is sent when pushing through Firebase, because these two are mutually exclusive. However, I feel we don't need to verify that. It's simply impossible to make a mistake: for anonymous messages, we only have the device ID and no mobile number. For direct messages, we only have the mobile number, no device ID.

Reply:
OK, removed this assertion